### PR TITLE
[ADD] module 'purchase_requisition_webkit'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Replaces the legacy rml Requisition report by a brand new webkit report.

I had some issues producing the translation file 'purchase_requisition_webkit.pot'. I had to build it using the approach described here: https://www.odoo.com/es_ES/forum/help-1/question/how-to-export-translation-strings-from-mako-templates-61575

### CODE REVIEW

File: purchase_requisition_webkit/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+#
+##############################################################################
+
+{
+    'name': 'Purchase Requisition Report using Webkit Library',
+    'version': '1.0.1',
+    'category': 'Reports/Webkit',
+    'description': """
+Replaces the legacy rml Requisition report by a brand new webkit report.""",
```
Comment (pedrobaeza) [inline purchase_requisition_webkit/__openerp__.py]: Put OCA's template here in description

File: purchase_requisition_webkit/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    'author': "Eficent,Odoo Community Association (OCA)",
+    'website': 'http://www.eficent.com',
+    'depends': ['base', 'report_webkit', 'base_headers_webkit',
```
Comment (pedrobaeza) [inline purchase_requisition_webkit/__openerp__.py:90]: Put that depends on _base_headers_webkit_ that it's on its repo

File: purchase_requisition_webkit/__openerp__.py (hunk 1 above)
Comment (JordiBForgeFlow) [inline purchase_requisition_webkit/__openerp__.py]: @pedrobaeza What do you mean by OCA's template? Could you give me an example?

Comment (pedrobaeza) [inline purchase_requisition_webkit/__openerp__.py]: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by JordiBForgeFlow** [comment]
Done! Thanks @pedrobaeza

**Comment by pedrobaeza** [comment]
Thanks for the change. One last thing: you don't need to put SVG file, because you're not customizing the icon. The file is provided on the template for someone who wants to customize it.

**Comment by JordiBForgeFlow** [comment]
Done! I also completed the html file.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**